src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderList.jsx

```diff
@@ -34,8 +35,12 @@ export default function ProviderList({
     sortMethod,
     typeOfCareName,
   } = useSelector(selectProviderSelectionInfo, shallowEqual);
+
+  const uniqueProviderList = communityCareProviderList
+    ? removeDuplicateId(communityCareProviderList)
```

Comment:
You might be able to move the removeDuplicateId call into `selectProviderSelectionInfo` and avoid a lot of the updates in the components themselves.

Reply:
Great call. So much simpler putting the removeDuplicateId call into `selectProviderSelectionInfo` than in the components.